### PR TITLE
Bunch of minor doc fixes

### PR DESCRIPTION
**Pull request by gerardsn**
Bunch of minor fixes.
See https://nuts-node.readthedocs.io/en/docs-minor-updates/index.html

### CODE REVIEW

File: docs/pages/deployment/database-configuration.rst [hunk 1]
```diff
@@ -75,7 +77,7 @@ Migrating to Vault
 
 Migrating your private keys from the filesystem to Vault is relatively easy: just upload the keys to Vault under ``kv/nuts-private-keys``.
 
-Alternatively you can use the ``fs2vault`` crypto command, which takes the directory containing the private keys as argument (the example assumes the container is called `nuts-node`):
+Alternatively you can use the ``fs2vault`` crypto command, which takes the directory containing the private keys as argument (the example assumes the container is called `nuts-node` and `NUTS_DATADIR=/opt/nuts/data`):
```
Comment (reinkrul) [inline docs/pages/deployment/database-configuration.rst:80]:
```suggestion
Alternatively you can use the ``fs2vault`` crypto command, which takes the directory containing the private keys as argument (the example assumes the container is called ``nuts-node`` and ``NUTS_DATADIR=/opt/nuts/data``):
```

File: docs/pages/deployment/monitoring.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ The main goal of the service is to give a YES/NO answer for if the service is ru
 
     GET /status
 
-It'll return an "OK" response and a 200 status code.
+Returns an "OK" response body wit status code 200.
```
Comment (reinkrul) [inline docs/pages/deployment/monitoring.rst:18]:
```suggestion
Returns an "OK" response body with status code ``200``.
```

File: docs/pages/deployment/monitoring.rst [hunk 3]
```diff
@@ -31,14 +31,26 @@ Basic diagnostics
 
     GET /status/diagnostics
 
-It'll return some text displaying the current status of the various services:
+Returns some text displaying the current status of the various services in ``yaml`` format:
```
Comment (reinkrul) [inline docs/pages/deployment/monitoring.rst:34]:
```suggestion
Returns the status of the various services in ``yaml`` format:
```

File: docs/pages/deployment/recommended-deployment.rst [hunk 4]
```diff
@@ -95,8 +95,11 @@ There are several endpoints that need to be accessed by external systems.
 You typically configure 2 subdomains for these, given `example.com` and the acceptance environment:
 
 * `nuts-acc.example.com` for traffic between nodes:
-  * HTTP traffic to `/n2n`
-  * gRPC traffic. gRPC will have to be bound on a separate port, e.g. `5555` (default).
+
+   * HTTP traffic to `/n2n`
```
Comment (reinkrul) [inline docs/pages/deployment/recommended-deployment.rst:99]:
```suggestion
   * HTTP traffic to ``/n2n``
```

File: docs/pages/deployment/recommended-deployment.rst [hunk 5]
```diff
@@ -95,8 +95,11 @@ There are several endpoints that need to be accessed by external systems.
...
+
+   * gRPC traffic. gRPC will have to be bound on a separate port, e.g. `5555` (default).
```
Comment (reinkrul) [inline docs/pages/deployment/recommended-deployment.rst:101]:
```suggestion
   * gRPC traffic, which will have to be bound on a separate port, e.g. ``5555`` (default).
```

File: docs/pages/deployment/recommended-deployment.rst [hunk 6]
```diff
@@ -95,8 +95,11 @@ There are several endpoints that need to be accessed by external systems.
...
+
 * `nuts-public-acc.example.com` for HTTP traffic to `/public`
```
Comment (reinkrul) [inline docs/pages/deployment/recommended-deployment.rst:103]:
```suggestion
* ``nuts-public-acc.example.com`` for HTTP traffic to ``/public``
```

File: docs/pages/development/3-api.rst [hunk 7]
```diff
@@ -78,7 +78,7 @@ All paths start with a security ``<context>``:
 - ``/n2n/**`` These APIs must be available to other nodes from the network.
   This means they must be protected with the required client certificate as specified by `RFC011 <https://nuts-foundation.gitbook.io/drafts/rfc/rfc011-verifiable-credential>`_.
   The creation of an access token is one example of such an API.
-- ``/public/**`` These APIs must be publicly available on a valid domain. No security must be set.
+- ``/public/**`` These APIs must be publicly available on a valid domain. Must set NO security other than a server certificate.
```
Comment (reinkrul) [inline docs/pages/development/3-api.rst:81]:
```suggestion
- ``/public/**`` These APIs must be publicly available on a valid domain. No security must be configured other than a server certificate.
```

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: docs/pages/deployment/database-configuration.rst (hunk 1 above)
Comment (gerardsn) [inline docs/pages/deployment/database-configuration.rst:80]:
```suggestion
Alternatively you can use the ``fs2vault`` crypto command, which takes the directory containing the private keys as argument (the example assumes the container is called *nuts-node* and *NUTS_DATADIR=/opt/nuts/data*):
```

File: docs/pages/deployment/tls-configuration.rst [hunk 8]
```diff
@@ -108,6 +108,11 @@ Configuration for `HAProxy <https://www.haproxy.com/>`_ could look like this (gi
 
 Refer to the HAProxy documentation for more information.
 
+.. note::
+
+    In a (level 4) pass-through configuration, the origin IP address for all connections to the Nuts node will be equal to that of the load balancer.
+    You will have to handle IP logging in this configuration yourself.
+
```
Comment (gerardsn) [inline docs/pages/deployment/tls-configuration.rst:115]: Can someone please verify this and is this sufficient documentation?

Comment (reinkrul) [reply docs/pages/deployment/tls-configuration.rst:115]: I don't think is true, but only applicable to TLS offloading?

Comment (gerardsn) [reply docs/pages/deployment/tls-configuration.rst:115]: I think this is true for all configurations. The IP address will be that of the reverse proxy.

The difference between lvl4 and lvl7 is that on lvl 7 you can add the `X-Forwarded-For` header so the node knows the actual origin. AFAIK there is no standard solution for this on lvl4. See [HAProxy docs](https://www.haproxy.com/documentation/hapee/2-4r1/load-balancing/client-ip-preservation/) on this as well.

**Review by reinkrul — APPROVED** [review]

**Pull request merged**